Vungle/VungleRouter.m

```diff
@@ -774,11 +775,15 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
                                                                               adMarkup:adMarkup];
     if (targetDelegate) {
         if (isAdPlayable) {
-            MPLogInfo(@"Vungle: Ad playability update returned ad is playable for Placement ID: %@", placementID);
             [targetDelegate vungleAdDidLoad];
         } else {
             MPLogInfo(@"%@", message);
-            if (!self.isAdPlaying) {
+            // Ignore any playability update if the delegate is the playing ad
+            // The SDK will fire playability updates during a successful playback
+            // to relay the status. We don't want to trigger the fail load if it
+            // is successfully playing. But don't block other placement load fails
+            if ([[self.playingFullScreenAdDelegate getPlacementID] isEqualToString:placementID] &&
+                [[self.playingFullScreenAdDelegate getAdMarkup] isEqualToString:adMarkup]) {
```

Comment:
Hi Stanley, we should not block other placement load fails, right? So might above judgment add `!` to check if the placementID is other placement? Thanks a lot. :)

Reply:
Nice catch